reference-ril/reference-ril.c

```diff
@@ -1603,7 +1603,7 @@ static void requestRegistrationState(int request, void *data,
         ALOGD("registration state type: 3GPP2");
         // TODO: Query modem
         if(request == RIL_REQUEST_VOICE_REGISTRATION_STATE) {
-            asprintf(&responseStr[3], "8");     // EvDo revA
+            asprintf(&responseStr[3], "%d", type);  // Radio Technology
```

Comment:
Please also add some comments about the qemu doesn't follow the spec in +CREG and +CGREG. Thank you.